File: generated-sources/scalaz/mojang-sessions/src/main/scala/com/github/asyncmc/mojang/sessions/scalaz/model/PlayerProfileProperty.scala
package com.github.asyncmc.mojang.sessions.scalaz.model

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import PlayerProfileProperty._

case class PlayerProfileProperty (
  /* The property name */
  name: String,
/* The serialized property value in base64. */
  value: ByteArray,
/* signed data using Yggdrasil's private key */
  signature: Option[ByteArray])

object PlayerProfileProperty {
  import DateTimeCodecs._

  implicit val PlayerProfilePropertyCodecJson: CodecJson[PlayerProfileProperty] = CodecJson.derive[PlayerProfileProperty]
  implicit val PlayerProfilePropertyDecoder: EntityDecoder[PlayerProfileProperty] = jsonOf[PlayerProfileProperty]
  implicit val PlayerProfilePropertyEncoder: EntityEncoder[PlayerProfileProperty] = jsonEncoderOf[PlayerProfileProperty]
}

File: generated-sources/php-slim/mojang-sessions/lib/com/github/asyncmc/mojang/sessions/php/slim/model/PlayerProfileProperty.php
<?php
/**
 * PlayerProfileProperty
 */
namespace com.github.asyncmc.mojang.sessions.php.slim.server\com.github.asyncmc.mojang.sessions.php.slim.model;

/**
 * PlayerProfileProperty
 */
class PlayerProfileProperty
{
    
    /** @var string $name The property name*/
    private $name;
    
    /** @var string $value The serialized property value in base64.*/
    private $value;
    
    /** @var string $signature signed data using Yggdrasil&#39;s private key*/
    private $signature;
}
